### PR TITLE
Profiles: don't normalize search input display text

### PR DESCRIPTION
**Pull request by benisgold**
Fixes RNBW-3201

## What changed (plus any additional context for devs)
- normalize ens behind the scenes only, keep raw user input displayed in search
- lint fixed something unrelated
- added horizontal inset to pending registration divider
- moved divider inside PendingRegistrations component so it's only shown when there are pending registrations

## PoW (screenshots / screen recordings)

https://user-images.githubusercontent.com/15272675/164295547-209a7608-6660-4800-90d0-9a713e065469.mp4

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?
- [ ] Added e2e tests? if not please specify why
- [ ] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

File: src/screens/ENSSearchSheet.js [hunk 1]
```diff
@@ -135,14 +135,7 @@ export default function ENSSearchSheet() {
...
           )}
-          {isIdle && (
-            <>
-              <Inset vertical="24px">
-                <Divider />
-              </Inset>
-              <PendingRegistrations />
-            </>
-          )}
+          {isIdle && <PendingRegistrations />}
```
Comment (estebanmino) [inline src/screens/ENSSearchSheet.js:138]: `isIdle` didn't make it? we talked with @jxom about this when building this feature and it was working

**Review by estebanmino — APPROVED** [review]
lgtm!

File: src/screens/ENSSearchSheet.js (hunk 1 above)
Comment (benisgold) [reply src/screens/ENSSearchSheet.js:138]: No, I can see if I can tweak it to make isIdle do the trick

**Comment by jxom** [comment]
Hmm, I'm not sure about this change.

Don't we want to normalize the input on change so it's consistent with checking if the normalized input is an available ENS?

**Comment by benisgold** [comment]
@jxom i thought that's what we decided on during our sync up, but you're right. gotta find another way

**Comment by benisgold** [comment]
i changed it back to how it was before (normalize the searchQuery immediately) but added a new variable that stores the raw, un-normalized input value

**Comment by jxom** [comment]
yo ben, we might want to have a look into https://github.com/adraffy/ens-normalize.js as well maybe maybe.

i'll approve this for now tho.

**Review by jxom — APPROVED** [review]

**Review by estebanmino — APPROVED** [review]

**Pull request merged**